src/Components/Product/Products.jsx

```diff
@@ -1,80 +1,46 @@
+import React, { useState, useEffect } from 'react'
+
 const Products = () => {
+    const [productsData, setProductsData] = useState([])
+
+    useEffect(() => {
+
+        (async () => {
+            try {
+                const response = await fetch('/api/products', { method: "GET" });
+                const json = await response.json()
+                setProductsData(json.products)
+            } catch (err) {
+                console.log(err)
```

Comment:
Remove this console.log if not needed or else you can display on the UI - the error message

Reply:
ok